### PR TITLE
[docs] Add basic context example

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
Issue: https://github.com/lit/lit.dev/issues/1183

### Context

For graduation we need some playground examples.
This is a "basic" example of using the Context package.

### Test plan

Tested manually

### CODE REVIEW

File: packages/lit-dev-content/samples/examples/context-basics/elements.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import {html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {provide, consume, createContext} from '@lit-labs/context';
+
+import {providerStyles} from './styles.js';
+
+const contextKey = Symbol('contextKey');
+// The value type for the context.
```
Comment (rictic) [inline packages/lit-dev-content/samples/examples/context-basics/elements.ts:8]: The word context is reaching semantic satiation here. context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context, context

How about we reword to give another angle on it.

```suggestion
// The values that will be provided and consumed in this demo will
// be strings.
```

File: packages/lit-dev-content/samples/examples/context-basics/elements.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+type ContextValue = string;
+// Context object, which acts like a key for the context.
+const context = createContext<ContextValue>(contextKey);
```
Comment (rictic) [inline packages/lit-dev-content/samples/examples/context-basics/elements.ts:11]: In a similar vein, let's inline let's inline the `contextKey` constant:

```suggestion
// The context object acts as a key. A consumer will only receive
// values from a provider if their contexts match. A Symbol ensures
// that this context will be unique.
const context = createContext<ContextValue>(Symbol());
```

**Review by rictic — APPROVED** [review]

File: packages/lit-dev-content/samples/examples/context-basics/elements.ts (hunk 2 above)
Comment (AndrewJakubowicz) [reply packages/lit-dev-content/samples/examples/context-basics/elements.ts:12]: Agree!

File: packages/lit-dev-content/samples/examples/context-basics/project.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "/samples/base.json",
+  "title": "Basics",
+  "description": "Share data from a provider element with Context.",
+  "section": "@lit-labs/context",
```
Comment (augustjk) [inline packages/lit-dev-content/samples/examples/context-basics/project.json:5]: wdyt about putting this under the section "Managing Data" to match the doc location? The title could be "Context Basics".

Comment (AndrewJakubowicz) [reply packages/lit-dev-content/samples/examples/context-basics/project.json:5]: I love that.

**Review by augustjk — APPROVED** [review]
Nice!

**Pull request merged**